tools/e2e-commons/package.json

```diff
@@ -14,35 +14,31 @@
 		"tunnel:reset": "./bin/tunnel.sh reset",
 		"tunnel:off": "./bin/tunnel.sh down",
 		"tunnel:logs": "pm2 logs --nostream --lines 10000 > output/logs/tunnel.log",
-		"config:decrypt": "openssl enc -md sha1 -aes-256-cbc -pass env:CONFIG_KEY -d -in ./config/encrypted.enc -out ./config/local-test.js",
-		"config:encrypt": "openssl enc -md sha1 -aes-256-cbc -pass env:CONFIG_KEY -in ./config/local-test.js -out ./config/encrypted.enc",
+		"config:decrypt": "openssl enc -md sha1 -aes-256-cbc -pass env:CONFIG_KEY -d -in ./config/encrypted.enc -out config/local.cjs",
+		"config:encrypt": "openssl enc -md sha1 -aes-256-cbc -pass env:CONFIG_KEY -in config/local.cjs -out ./config/encrypted.enc",
 		"preinstall": "pnpm --prefix ../cli install",
+		"postinstall": "pnpm playwright install",
 		"jetpack-connect": "NODE_ENV=test pnpm exec babel-node bin/e2e-jetpack-connector.js"
 	},
 	"devDependencies": {
-		"@babel/core": "7.16.0",
-		"@babel/preset-env": "7.16.4",
-		"@babel/node": "7.16.0",
+		"@playwright/test": "1.16.3",
 		"@slack/web-api": "6.4.0",
 		"@wordpress/eslint-plugin": "9.3.0",
+		"allure-playwright": "2.0.0-beta.14",
 		"axios": "0.24.0",
 		"chalk": "4.1.2",
 		"config": "3.3.6",
 		"eslint": "7.32.0",
-		"eslint-plugin-jest": "25.2.4",
-		"jest": "27.3.1",
-		"jest-circus-allure-environment": "1.1.0",
+		"eslint-plugin-playwright": "0.6.0",
 		"jetpack-cli": "link:../cli",
 		"localtunnel": "2.0.2",
 		"lodash": "4.17.21",
-		"mkdirp": "1.0.4",
-		"playwright": "1.16.3",
 		"pm2": "5.1.2",
-		"rimraf": "3.0.2",
 		"shell-escape": "0.2.0",
 		"winston": "3.3.3",
 		"wpapi": "1.2.2",
-		"yargs": "17.2.1"
+		"yargs": "17.2.1",
+		"prettier-plugin-svelte": "2.5.0"
```

Comment:
🤔 Why do we need it here?

Reply:
Oh, I forgot about this. There was an eslint error caused by some version bump, complaining about missing `prettier-plugin-svelte` dependency. It was my way of not dealing with it and moving on with my PR. I'll remove it.

Reply:
Gotcha. I think boost is using svelte, so it might have been some uninstalled dependencies from boost plugin

Reply:
It looks like this appeared after a bump of @wordpress/eslint-plugin. I downgraded back to 9.2.0 for now, just to get this out and I'll follow up on this.